x/withdraw/keeper/ibc_callbacks_test.go

```diff
@@ -0,0 +1,292 @@
+package keeper_test
+
+import (
+	"fmt"
+
+	"github.com/cosmos/cosmos-sdk/crypto/keys/secp256k1"
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	authtypes "github.com/cosmos/cosmos-sdk/x/auth/types"
+	"github.com/stretchr/testify/mock"
+	"github.com/tharsis/ethermint/crypto/ethsecp256k1"
+	"github.com/tharsis/evmos/v2/testutil"
+
+	transfertypes "github.com/cosmos/ibc-go/v3/modules/apps/transfer/types"
+	clienttypes "github.com/cosmos/ibc-go/v3/modules/core/02-client/types"
+	channeltypes "github.com/cosmos/ibc-go/v3/modules/core/04-channel/types"
+	ibcgotesting "github.com/cosmos/ibc-go/v3/testing"
+	ibcmock "github.com/cosmos/ibc-go/v3/testing/mock"
+
+	claimstypes "github.com/tharsis/evmos/v2/x/claims/types"
+	vestingtypes "github.com/tharsis/evmos/v2/x/vesting/types"
+	"github.com/tharsis/evmos/v2/x/withdraw/keeper"
+	"github.com/tharsis/evmos/v2/x/withdraw/types"
+)
+
+func (suite *KeeperTestSuite) TestOnRecvPacket() {
+	// secp256k1 account
+	secpPk := secp256k1.GenPrivKey()
+	secpAddr := sdk.AccAddress(secpPk.PubKey().Address())
+	secpAddrEvmos := secpAddr.String()
+	secpAddrCosmos := sdk.MustBech32ifyAddressBytes(sdk.Bech32MainPrefix, secpAddr)
+
+	// ethecp256k1 account
+	ethPk, err := ethsecp256k1.GenerateKey()
+	suite.Require().Nil(err)
+	ethsecpAddr := sdk.AccAddress(ethPk.PubKey().Address())
+	ethsecpAddrEvmos := sdk.AccAddress(ethPk.PubKey().Address()).String()
+	ethsecpAddrCosmos := sdk.MustBech32ifyAddressBytes(sdk.Bech32MainPrefix, ethsecpAddr)
```

Comment:
er, this is just an invalid address yes? Isn't it better to specify is it an invalid address rather than `ethsecpAddrCosmos`? Easier for non-implementer of the code to understand